libs/net/dal/Migrations/1.0.144/Up/PostUp/00-NotificationTemplate.sql

```diff
@@ -0,0 +1,55 @@
+DO $$
+BEGIN
+
+-- Update all notification templates.
+UPDATE public."notification_template" SET
+    "body" = '@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.TemplateEngine.Models.Notifications.NotificationEngineContentModel>
+@using System
+@{
+  var subscriberAppUrl = @SubscriberAppUrl?.ToString();
+  var viewContentUrl = @ViewContentUrl?.ToString();
+  var requestTranscriptUrl = @RequestTranscriptUrl?.ToString();
+  var addToReportUrl = @AddToReportUrl?.ToString();
+  var isAV = @Content.ContentType == TNO.Entities.ContentType.AudioVideo;
+  var isTranscriptAvailable = isAV && !string.IsNullOrWhiteSpace(@Content.Body) && @Content.IsApproved;
+  var sourceCode = !string.IsNullOrEmpty(@Content.Source?.Code) ? @Content.Source.Code : @Content.OtherSource;
+  var body = isTranscriptAvailable ? @Content.Body : (isAV ? @Content.Summary : @Content.Body);
+  //  body = System.Text.RegularExpressions.Regex.Replace(body, @"\r\n?|\n", "<br/>");
+  var replaceWith = "<br/>";
+  body = body.Replace("\r\n", replaceWith).Replace("\n", replaceWith).Replace("\r", replaceWith);
+
+  var tz = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
+  var utcOffset = tz.GetUtcOffset(System.DateTime.Now).Hours;
+}
+<div>@sourceCode (@Content.Source?.Name)</div>
+@if (!string.IsNullOrEmpty(@Content.Series?.Name))
+{
+  <div>@Content.Series.Name</div>
+}
+<div>@Content.PublishedOn?.AddHours(@utcOffset).ToString("dd-MMM-yyyy")</div>
+@if (!string.IsNullOrEmpty(@Content.Byline))
+{
+  <div>@Content.Byline</div>
+}
+<br/>
+<div>@body</div>
+<br />
+@if (!string.IsNullOrEmpty(subscriberAppUrl))
+{
+  <div><a href="@subscriberAppUrl" target="">MMI</a> :: <a href="@viewContentUrl@Content.Id" target="_blank">View Article</a></div>
+  <br />
+}
+@if (isAV && !isTranscriptAvailable && !string.IsNullOrEmpty(requestTranscriptUrl) && Content.Source?.DisableTranscribe == false)
+{
+  <div><a href="@requestTranscriptUrl@Content.Id" target="_blank">Request Transcript...</a></div>
+  <br />
+}
+<br />
```

Comment:
Removed Add to Report link as it's out of scope and difficult to implement this way.